src/signals/incident-management/components/FilterForm/actions.js

```diff
@@ -19,6 +20,11 @@ export const setAddress = payload => ({
   payload,
 });
 
+export const setNoteKeyword = payload => ({
```

Comment:
Test missing for this function

Reply:
Sorry, should have run coverage, working on it!